pages/news/2020-05-12-mixxx-2-3-beta-released.html

```diff
@@ -0,0 +1,84 @@
+title: Mixxx 2.3 beta released
+author: Be.
+date: 2020-05-12 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+We are pround to announce the release of Mixxx 2.3 beta, [download it!](/download/) This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more. A brief overview of the changes is below. In the coming weeks, we will publishing a series of posts on this blog discussing the new features in more depth, so stay tuned!
+
+<img src="{% static '/static/images/news/screenshot-2.3-beta-deck.png' %}" width="633" height="275"/>
```

Comment:
`<img>` tags need an `alt` attribute:

```suggestion
<img src="{% static '/static/images/news/screenshot-2.3-beta-deck.png' %}" width="633" height="275" alt="Screenshot of a Mixxx deck with new waveform markers" />
```

The image does not reflect the current state of LateNight in master. Can you make a new one? Maybe even use the LateNight Facelift Pt. 3 branch (only if we merge that soon).

Reply:
I don't know what you mean. This screenshot was taken with the current state of LateNight in the 2.3 branch. When the LateNight Facelift Pt 3 branch is merged we can update the screenshot.

I added the alt attribute.

Reply:
I guess I spent too much time on the LateNight Facelift III branch already and my memory of the current LateNight started to blur :smile: sorry for the noise.